backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepository.java

```diff
@@ -77,6 +77,10 @@ public UUID createShop(CreateShopRequest createShopRequest) {
 				.map(Map.Entry::getValue).collect(Collectors.toList());
 	}
 
+	public Long getWaitingTime() {
+		return 2L;
```

Comment:
should we have a variable defined in top and return that variable instead, rather than a value? Just a suggestion.

Reply:
Sure done.